### PR TITLE
Add option to install devcontainer.json

### PR DESCRIPTION
**Pull request by GDYendell**
- This allows vscode to open the directory inside the dev-c7 container

### CODE REVIEW

File: run-dev-c7.sh [hunk 1]
```diff
@@ -69,6 +72,14 @@ done
 
 shift $((OPTIND-1))
 
+if ${install} ; then
+    mkdir -p .devcontainer
+    cd .devcontainer
+    wget -q https://github.com/dls-controls/dev-c7/releases/download/2.1.0/devcontainer.json
```
Comment (gilesknap) [inline run-dev-c7.sh:78]: I believe you need to use ${version} instead of 2.1.0

Comment (GDYendell) [reply run-dev-c7.sh:78]: Yeah I wanted to do that, but `https://github.com/dls-controls/dev-c7/releases/download/latest/devcontainer.json` does not exist. There are only specific versions releases assets, as far as I can tell.

Comment (gilesknap) [reply run-dev-c7.sh:78]: In the CI the 'latest' is replaced with current version and then that is the published file.

And $version is set to that (replaced) latest in the top of the script 
https://github.com/dls-controls/dev-c7/blob/cb4c655990c3a14dd9dc736e59e68b72e6c0f184/run-dev-c7.sh#L8

https://github.com/dls-controls/dev-c7/blob/cb4c655990c3a14dd9dc736e59e68b72e6c0f184/.github/workflows/code.yml#L78-L93

Comment (GDYendell) [reply run-dev-c7.sh]: Ah, fancy. OK fixed.

**Pull request merged**